start-deployCF

```diff
@@ -91,7 +91,9 @@ if ! isTrue ${USE_MODPACK_START_SCRIPT:-true}; then
     echo "${FTB_SERVER_MOD}" > $installMarker
   fi
 
-  export SERVER=$(find ${FTB_BASE_DIR} -type f \( -path "*/libraries/*" -o -path "*/mods/*" \) -prune -o -name "forge*.jar" -not -name "forge*installer.jar" -print)
+  # change from */mods/* to /mods/* as leading * makes things fail
+  # and add maxdepth 2
```

Comment:
You can remove these inline comments explaining the change since the PR will provide that history.